liferay-frontend/liferay-portal
# LPS-136418 Insert button appears when shifting a paragraph

JIRA: https://issues.liferay.com/browse/LPS-136418

To test:
1. Place `frontend-editor-ckeditor-sample-web` on a page
2. See gif below

The root cause of the issue was incomplete set of conditions when not a render the insert button. In this PR, we are replacing this set when not to render with a single condition when to render: an empty paragraph. The solution is not ideal, as it somewhat depends on the implementation of CKEditor, but is also true for non-render conditions. Conversely, I tried to check that the selection element is `p` for paragraph, but `h*` can also be used, for heading text styles.

In addition to main purpose of this PR, in this PR we are:
- Fixing RTL positioning (see gifs)
- Fixing vertical positioning of button and offseting it a bit to the side (see gifs)
- Improving button styles a bit
- Explicitly setting panel wrapper in plugin, instead of relying on `CKEDITOR.env.cssClass`. This class was not meant for this, but rather for global environment class setting, like `ie` or `mobile`. As such, is leaks into many unintended parts of editors, that may cause weird effects down the line. This simplifies the code as we don't need to store and recover it. Also, it fixes a minor annoying issue where `title` attribute is set to nothing, resulting in empty tooltips in some edge cases.
- Removing panel padding (`DEFAULT_PANEL_PADDING`) from global scope, there is no need to be in it.
- Cleaning up sample module from unnecessary bits.

Before PR:
![before](https://user-images.githubusercontent.com/5435511/127887273-c28cd29b-a0db-485d-90ef-eec40825578b.gif)

After PR:
![after](https://user-images.githubusercontent.com/5435511/127887250-e16550ea-a664-46f2-858a-13c62d20ffd9.gif)
